README.md

```diff
@@ -2,6 +2,8 @@
 
 Phimp.me is a Photo App for Android that aims to replace proprietary photo applications. It offers features such as taking photos, adding filters, editing images and uploading them to social networks.
 
+Phimp.me is a Photo App for Android that aims to replace proprietary photo applications. It offers features such as taking photos, adding filters, editing images and uploading them to social networks.
```

Comment:
Duplicate text please see the above paragraph or point. Is'nt this is an unwanted addition?